apps/vs-code-intellisense/package.json

```diff
@@ -71,7 +71,7 @@
     "test": "node ./out/test/runTest.js"
   },
   "dependencies": {
-    "@twilio-paste/design-tokens": "^8.5.0",
+    "@twilio-paste/design-tokens": "^9.0.2",
```

Comment:
Hmmmm internal dependency versioning should be handled by changesets. I wonder if this is set up correctly as workspace? If it's not detected as a workspace package it won't get version updates.

Publishing to the Microsoft extension registry is a whole other thing

Reply:
The VS Marketplace is definitely a little clunky.

Do you have an example of how this dependency should be setup? 🤔